src/server/list/list.model.ts

```diff
@@ -22,9 +25,26 @@ export type UsableList = {
 	phoneNumber: string | null;
 	eventDate: string | null;
 	comment: string | null;
-	submitted: boolean;
-	verified: boolean;
+	status: ListStatus;
+	submittedAt: Date | null;
+	verified: ListVerification | null;
+	// submitted: boolean;
+	// verified: boolean;
 	createdBy: MinimalUser;
+	ownedBy: MinimalUser;
+};
+
+export type IncompleteMinimalList = Omit<MinimalList, 'status'>;
+
+export type IncompleteUsableList = Omit<UsableList, 'ownedBy' | 'status'> & {
+	ownedBy: MinimalUser | null;
+};
+
+export type ListStatus = 'open' | 'submitted' | 'verified';
```

Comment:
Perhaps enum in TS is better for this type of variable?

Reply:
I tend to prefer unions of string literals to enums in typescript since they provide the same level of type safety and IntelliSense, doesn't require imports and provides less bloat in compiled code. For this project in specific Prisma also converts database Enums into unions of string literals, so I think it makes sense to do it this way for consistency should we ever use enums in the database.